src/components/slide/slide.js

```diff
@@ -76,10 +76,18 @@ const AnimatedDiv = styled(animated.div)`
   height: 100%;
   position: absolute;
   background: transparent;
+  &:active {
+    outline: 1px solid white;
+  }
   ${({ tabIndex }) =>
     tabIndex === 0 &&
     css`
-      &:focus {
+      outline: 2px solid white;
+    `}
+  ${({ isOverviewMode }) =>
+    isOverviewMode &&
+    css`
+      &:hover {
```

Comment:
created `isOverviewMode` boolean to fix small bug where the hover styling was still being applied when a user was in DEFAULT_MODE:
![spectacle-hover-fix](https://user-images.githubusercontent.com/16032631/99859002-aa9ab180-2b43-11eb-97d9-40d06c1a4905.gif)